website/docs/toolchainversion.md

```diff
@@ -17,13 +19,19 @@ toolchainversion ("value")
 * `3.8`: Clang 3.8
 * `5.0`: Clang 5.0
 
+### Linux Projects ###
+
+* `remote`: Remote Compilation and Debugging
+* `wsl`: Windows Subsystem for Linux
+* `wsl2`: Windows Subsystem for Linux 2
+
 ## Applies To ###
 
 The `config` scope.
 
 ### Availability ###
 
-Premake 5.0.0 alpha 14 or later, only applies to Android projects.
+Premake 5.0.0 alpha 14 or later, only applies to Android or Linux projects.
```

Comment:
Split this into two statements. One for Android, one for Linux projects (describing correct versions for each)